top/cohttp_top.ml

```diff
@@ -0,0 +1,20 @@
+let printers = [ "Cohttp.Header.pp_hum"
```

Comment:
There's a much simpler scheme to do this which is to write the installation bits in a source `PKG_top_init.ml` file and simply have a library `PKG_top` with a call to `Toploop.use_file`. The advantage is that it's easier to test that the init works before making a release if you wish, you get good error reporting and you don't `sprintf` code which is kind of ugly. 

You can have a look at [`gg_top`](https://github.com/dbuenzli/gg/blob/master/src/gg_top.ml) and [`gg_top_init.ml`](https://github.com/dbuenzli/gg/blob/master/src/gg_top_init.ml) or [`mtime_top`](https://github.com/dbuenzli/mtime/blob/master/src-os/mtime_top.ml) and [`mtime_top_init.ml`](https://github.com/dbuenzli/mtime/blob/master/src-os/mtime_top_init.ml).